src/README.md

```diff
@@ -56,6 +56,16 @@ report it on the [issue tracker].
 [this repository]: https://github.com/rust-embedded/discovery
 [issue tracker]: https://github.com/rust-embedded/discovery/issues
 
+## Other embedded Rust resources
+
+This Discovery book is just one of several embedded Rust resources provided by the
+[embedded Working Group]. The full selection can be found at ["The embedded Rust Bookshelf"]. This
+includes the list of [Frequently Asked Questions].
+
+[embedded Working Group]: https://github.com/rust-embedded/wg
```

Comment:
I find the casing of the word embedded a bit weird. Any reason not to use uppercase here...

Reply:
I didn't mean that embedded should in general be spelled with capital 'E' but in the context of these links it looks a bit weird that everything is capitalised *except* for "embedded".

Reply:
The two uses that you have flagged are spelled with that lowercase spelling on that page:
> Welcome to an overview of the documentation provided by the embedded Working Group. 

...
> Many of these resources take the form of "books"; we collectively call these "The embedded Rust Bookshelf".

...
> Also known as "the book" by the embedded Rust community. The embedded Rust book will ...

Reply:
Any suggestions how to normalise this in a sensible way? We can also defer this, it just stuck out while reviewing the PR.

Reply:
Looks fantastic.

Reply:
Cool! I have pushed up that update to the PR.

Thanks for reviewing!